packages/ui-extensions/src/surfaces/customer-account/components/Menu/Menu.ts

```diff
@@ -0,0 +1,11 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import {IdProps} from '../../../checkout';
+
+export interface MenuProps extends IdProps {
```

Comment:
@oleksandroliynyk the doc won't show `id` as a prop despite extending `IdProps`.Avatar also has this problem, progress does not 🤔 Wanna pair on it someday? 

I don't think that's a blocker for this PR.